src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/adaptive_pooling_gpu_ref.cl

```diff
@@ -95,8 +99,14 @@ KERNEL(adaptive_pooling_gpu)(
 #endif
 
 #if MAX_POOLING
+    #if OUTPUT_DIMS == 5
```

Comment:
shouldn't be Dims = 6 support for this operation?

Reply:
No. According to https://docs.openvino.ai/latest/openvino_docs_ops_pooling_AdaptiveMaxPool_8.html , input is 3D/4D/5D.